AI-and-Analytics/Getting-Started-Samples/Modin_Vs_Pandas/README.md

```diff
@@ -1,16 +1,16 @@
-# `Intel® Modin* Vs. Pandas Performance` Sample
+# Modin Vs. Pandas Performance Sample
```

Comment:
README template should be applied to this sample README ([ONSAM-1958](https://jira.devtools.intel.com/browse/ONSAM-1958))